### DISCUSSION TITLE
群晖新套件：阿里云盘WebDAV 可挂载和同步 支持DSM6、7--被封禁？

### DISCUSSION THREAD

**Discussion by iYuXiaoyan** [messense/aliyundrive-webdav#830] [Q&A]
###  硬件：
-  白群晖DS218，DSM 6.2.4-25556
### 过程：
买了阿里云盘8T的会员；想把自己的群晖NAS上的文件备份到阿里云盘上。全程按照——[群晖新套件：阿里云盘WebDAV 可挂载和同步 支持DSM6、7](https://imnks.com/3939.html)进行的操作，确定步骤没有问题，但是报错了【无法连接到服务器，请确认输入的信息正确……】。见下图。

![](https://pic2.imgdb.cn/item/645f4b440d2dde5777aa939b.png)

我上面ip是我本地的群晖的ip，请问，是不是阿里云盘不支持第三个这么访问了？后面该如何处理？

**Comment by ocean-zhc** [comment]
不会，支持第三方访问，我是在OpenWrt安装的服务。
<img width="646" alt="image" src="https://github.com/messense/aliyundrive-webdav/assets/46189785/d27f0597-e2a8-475a-88e4-a4d2cfaa2a1c">

**Comment by dont-see-big-shark** [comment]
套件的bug
